src/components/cards/shared-card.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Thumbnail } from '../../interfaces/thumbnail';
+import { PillCategory } from '../../types/pill-category';
+import { TypographyScale } from '@mdb/flora';
+import {
+    SharedCardWrapper,
+    ThumbnailImage,
+    StyledPill,
+    FooterContent,
+    ThumbnailWrapper,
+    StyledHorizontalRule,
+    StyledTitle,
+    StyledDescription,
+    IntrinsicRatioWrapper,
+} from '../../styled/shared-card';
+
+//It will consume prop support for no thumbnail, large-medium-small thumbnail, pill, title, description, a footer with date
+interface IProps {
+    contentDate: Date;
+    description?: string;
+    title: string;
+    pillCategory: PillCategory;
+    tags?: string[];
+    thumbnail?: Thumbnail;
+}
+
+const thumbnailSizeMap = {
+    small: { height: '64px', width: '64px' },
+    medium: { height: '96px', width: '96px' },
+    large: { height: '184px', width: '327px' },
+};
+
+const myLoader = ({ src, width, quality }: any) => {
+    return `${src}?w=${width}&q=${quality || 75}`;
+};
+
+const SharedCard: React.FunctionComponent<IProps> = ({
+    contentDate,
+    description,
+    title,
+    pillCategory,
+    thumbnail,
```

Comment:
since we are only referencing the `url` and `size` property from the `thumbnail` any chance we can just explicitly reference the thumbnail `url` and `size`  by having a prop named `thumbnailUrl`  and thumbnailSize or we could destructure it?